app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
```

Comment:
- [ ] **[OPTIONAL]** I believe why the previous reviewer was having this issue using the app is because you did not add the `authenticate_user!` helper method in the application controller file. Users who are not logged in should be automatically redirected to the sign-in page

  ```
  before_action :authenticate_user!
  ```

Reply:
Hey, @julie-ify thanks a lot for the tip! I just added it. 😄